packages/plugins/i18n/server/constants/iso-locales.json

```diff
@@ -448,13 +456,21 @@
     "name":"English (en)"
   },
   {
-    "code":"en-AS",
-    "name":"English (American Samoa) (en-AS)"
+    "code":"en-AI",
+    "name":"English (Anguilla) (en-AI)"
+  },
+  {
+    "code":"en-AG",
+    "name":"English (Antigua & Barbuda) (en-AG)"
```

Comment:
@aeadedoyin this one can be removed since the language one you added should cover it: 
```
  {
    "code":"aig",
    "name":"Antigua and Barbuda Creole English"
  },
```

Reply:
@ronronscelestes I have updated this.

Reply:
thank you! 
tests are running, waiting for them to pass then we can merge your PR today so it'll be included in tomorrow's release :)